Index: ncurses/tinfo/lib_baudrate.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/ncurses/tinfo/lib_baudrate.c b/ncurses/tinfo/lib_baudrate.c
--- a/ncurses/tinfo/lib_baudrate.c	(revision 87c2c84cbd2332d6d94b12a1dcaf12ad1a51a938)
+++ b/ncurses/tinfo/lib_baudrate.c	(date 1730160342798)
@@ -79,7 +79,11 @@
 #undef B460800
 #undef B921600
 #define USE_OLD_TTY
+#ifdef __APPLE__
+#if !TARGET_OS_IPHONE
 #include <sys/ttydev.h>
+#endif
+#endif
 #else
 #undef USE_OLD_TTY
 #endif /* USE_OLD_TTY */

